hack/tools/tilt-prepare/main.go

```diff
@@ -353,8 +353,6 @@ func kustomizeTask(path, out string) taskFunction {
 			kustomizePath,
 			"build",
 			path,
```

Comment:
I had to add that so it works with kustomize v4.4.

Did you delete the kustomize binary so the new v4.4 binary is downloaded?